src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
+/* eslint-disable jsx-quotes */
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className='App'>
+    <Calculator />
+  </div>
+);
```

Comment:
- [ ] Please, consider that the App.js component should be class-based, as it is requested in this milestone 👍